src/Compilers/Core/VBCSCompiler/CSharpCompilerServer.cs

```diff
@@ -15,26 +17,27 @@ internal CSharpCompilerServer(string[] args, string clientDirectory, string base
         {
         }
 
-        public static int RunCompiler(
+        public static BuildResponse RunCompiler(
             string clientDirectory,
             string[] args,
             string baseDirectory,
             string sdkDirectory,
             string libDirectory,
             IAnalyzerAssemblyLoader analyzerLoader,
-            TextWriter output,
-            CancellationToken cancellationToken,
-            out bool utf8output)
+            CancellationToken cancellationToken)
         {
             var compiler = new CSharpCompilerServer(args, clientDirectory, baseDirectory, sdkDirectory, libDirectory, analyzerLoader);
-            utf8output = compiler.Arguments.Utf8Output;
+            bool utf8output = compiler.Arguments.Utf8Output;
 
-            foreach (var analyzer in compiler.Arguments.AnalyzerReferences)
+            if (!AnalyzerConsistencyChecker.Check(baseDirectory, compiler.Arguments.AnalyzerReferences, analyzerLoader))
```

Comment:
@mavasani I'm not clear on whether it is the responsibility of the csc.exe process that calls vbcscompiler.exe to show the logo banner, or the responsibility of the compilation running within vbcscompiler.exe to show it.